video/agon.h

```diff
@@ -26,7 +26,7 @@
 #define UART_RTS				13		// The ESP32 RTS pin (eZ80 CTS)
 #define UART_CTS	 			14		// The ESP32 CTS pin (eZ80 RTS)
 
-#define COMMS_TIMEOUT			100		// Timeout for VDP commands (ms)
+#define COMMS_TIMEOUT			200		// Timeout for VDP commands (ms)

```

Comment:
experimenting has shown the 100ms timeout wasn't _quite_ long enough, and that on some rare occasions (almost certainly to do with reading from SD cards) the z80 would delay beyond this limit